hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.tsx

```diff
@@ -465,6 +467,14 @@ export const PendingTable = () => {
                                 </TableBody>
                             </Table>
                         </TableContainer>
+                        <TableRow className={styles.title} style={{ float: "right" }}>
```

Comment:
Instead of float right (float is very bad, shamed upon in css community 😂) try wrapping it in a tablecell component aligned right. And if that doesn't work you can do some more googling